SofaKernel/modules/SofaBaseTopology/SparseGridTopology.cpp

```diff
@@ -1419,8 +1426,8 @@ void SparseGridTopology::buildVirtualFinerLevels()
     _virtualFinerLevels[0]->_fillWeighted.setValue( _fillWeighted.getValue() );
     _virtualFinerLevels[0]->init();
 
-    sout<<"SparseGridTopology "<<getName()<<" buildVirtualFinerLevels : ";
-    sout<<"("<<newnx<<"x"<<newny<<"x"<<newnz<<") -> "<< _virtualFinerLevels[0]->getNbHexahedra() <<" elements , ";
+    msg_info()<<"SparseGridTopology "<<getName()<<" buildVirtualFinerLevels : " << msgendl;
```

Comment:
There is no needs for msgendl at he end of message it is only to separate lines within a message.